bucket/vesta.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+    "version": "3.5.7",
+    "description": "VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.",
```

Comment:
Mentioning application name is not needed.

Reply:
description updated